test/e2e/data_setup/schema/recordset/product-recordset.json

```diff
@@ -825,7 +869,165 @@
           "display": "Categories"
         },
         "tag:isrd.isi.edu,2016:visible-columns": {
-          "*": ["id", "term"]
+          "*": ["id", "term", "type"]
+        },
+        "tag:isrd.isi.edu,2016:visible-foreign-keys" : {
```

Comment:
This annotation is unnecessary. We currently only use this annotation to control the list of related tables in record app.